frontend/src/components/filterable-table.js

```diff
@@ -184,15 +190,52 @@ function FilterableTable(props) {
     if (selected == null) {
         tableComponent = <ReactTable columns={columns} data={filteredData} />;
     }
+
+    function removeFilterString(filterString) {
+        const updatedFilterStrings = filterStrings.filter(
+            (oldString) => oldString !== filterString
+        );
+        setFilterStrings(updatedFilterStrings);
+    }
+
+    function addFilterString(filterString) {
+        const formattedFilterString = filterString.toLowerCase();
```

Comment:
It will confuse the user to see the strings become lower case when they are displayed. Instead, lower case them before they are used to filter, but display what they typed.

Reply:
OK, good point